### PR TITLE
GFF Sheek Bakrii Saphaloo Qubee Keyboard v1.0.1

### LINKED ISSUE #2482: bug(gff_sbs): missing pdf in documentation
gff_sbs/1.0.0/SBS-Typing-Manual.pdf is referenced but actually called SBS-Typing-English.pdf.

Detected in checks on https://github.com/keymanapp/help.keyman.com/pull/903

@dyacob

### PR DESCRIPTION
**Pull request by dyacob**
This PR fixes #2482 .

The `welcome.htm` and `gff_sbs.php` files both have the needed link corrections.  The correction was tested successfully on the `welcome.htm` page.

@DavidLRowe's earlier suggestion for the `README.md` file have also been applied.  Thanks @DavidLRowe !

### CODE REVIEW

**Comment by keyman-server** [comment]
This pull request is from an external repo and will not automatically be built. The build must still be passed before it can be merged. Ask one of the team members to make a manual build of this PR.

**Comment by LornaSIL** [comment]
Right now the version is 1.0.1 in several places (HISTORY.md and .kps), but in the kmn and .kpj it says 1.0.2. Can you set the version you want...probably the .kmn needs changing.

**Comment by dyacob** [comment]
@LornaSIL got it now, the result of another experiment that led to a bug report.  Thanks.

**Review by LornaSIL — APPROVED** [review]
LGTM

File: experimental/gff/gff_sbs/gff_sbs.kpj
Comment (zenodotus) [inline experimental/gff/gff_sbs/gff_sbs.kpj:1]: 'Athrinka' should be 'Athinkra'.

**Comment by LornaSIL** [comment]
Thanks. I created an issue so we can track it there since this PR is closed: #2490

**Pull request merged**